### PR TITLE
Updated helm chart for muster api

### PR DESCRIPTION
**Pull request by anitha-srinivasan-egov**

### CODE REVIEW

File: deploy-as-code/helm/environments/works-dev.yaml [hunk 1]
```diff
@@ -205,6 +205,7 @@ project-management-system:
 muster-roll-service:
   tracing-enabled: true
   java-enable-debug: true
+  app-timezone: "Asia/Kolkata"
```
Comment (pintu-eGov) [inline deploy-as-code/helm/environments/works-dev.yaml:208]: Please use the already defined timezone key (https://github.com/egovernments/DIGIT-DevOps/blob/c43646aa0bfa0f770ac64025b0d0449c3542ff41/deploy-as-code/helm/environments/works-dev.yaml#L60).

File: deploy-as-code/helm/charts/digit-works/backend/muster-roll-service/values.yaml [hunk 2]
```diff
@@ -77,6 +77,10 @@ env: |
     value: "save-musterroll"
   - name: MUSTERROLL_KAFKA_UPDATE_TOPIC
     value: "update-musterroll"
+  {{- if index .Values "app-timezone" }}
```
Comment (pintu-eGov) [inline deploy-as-code/helm/charts/digit-works/backend/muster-roll-service/values.yaml:80]: As per key name (i.e : timezone) change the corresponding condition here

File: deploy-as-code/helm/environments/works-dev.yaml (hunk 1 above)
Comment (anitha-srinivasan-egov) [reply deploy-as-code/helm/environments/works-dev.yaml]: Removed this from works-dev.yaml and made necessary change in values.yaml

File: deploy-as-code/helm/charts/digit-works/backend/muster-roll-service/values.yaml (hunk 2 above)
Comment (anitha-srinivasan-egov) [reply deploy-as-code/helm/charts/digit-works/backend/muster-roll-service/values.yaml]: Updated the values.yaml to read from the key timezone

**Review by pintu-eGov — APPROVED** [review]

**Pull request merged**